kaggle_environments/envs/lux_ai_2022/luxai2022/env.py

```diff
@@ -211,23 +211,35 @@ def _handle_bid(self, actions):
                     highest_bid = bid
                     highest_bid_agent = k
                 elif bid == highest_bid:
-                    # no one gets the extra factory
-                    highest_bid_agent = "tie"
+                    # if bids are the same, player 0 defaults to the winner and pays thebid.
+                    highest_bid_agent = "player_0"
             else:
                 # team k loses
                 failed_agents[k] = True
-        if highest_bid_agent == "tie":
-            pass
-        elif highest_bid_agent is None:
+        if highest_bid_agent is None:
             # no valid bids made, all agents failed.
             pass
         else:
             self.state.teams[highest_bid_agent].init_water -= highest_bid
             self.state.teams[highest_bid_agent].init_metal -= highest_bid
-            self.state.teams[highest_bid_agent].factories_to_place += 1
+            self.state.teams[highest_bid_agent].place_first = True
         return failed_agents
```

Comment:
Players might want to bid to go 2nd. Either need to add a second val to the bid to show go first or second, or allow "negative" bid values which means "I want to go second" the player with highest absolute value wins and gets their desired placement position.